testing/wait_for_backend_and_run_e2etests.sh

```diff
@@ -20,7 +20,7 @@ is_infrastructure_up () {
 export -f is_infrastructure_up
 ./gradlew :dataland-e2etests:compileTestKotlin :dataland-frontend:generateAPIClientFrontend :dataland-frontend:build --no-daemon --stacktrace
 timeout 240 bash -c "while ! is_infrastructure_up; do echo 'infrastructure not yet completely there - retrying in 1s'; sleep 1; done; echo 'infrastructure up!'"
-./gradlew :dataland-e2etests:test :dataland-frontend:generateAPIClientFrontend :dataland-frontend:npm_run_testpipeline --no-daemon --stacktrace
+./gradlew :dataland-e2etests:test dataland-frontend:generateAPIClientFrontend :dataland-frontend:npm_run_testpipeline --no-daemon --stacktrace
```

Comment:
Ist mir nicht klar warum der ":" hier weg muss, selbst wenn es funktioniert erscheint es mir inkonsistent mit den anderen Aufrufen in der gleichen Zeile

Reply:
Ich meine mich erinnern zu können, dass der versehentlich gelöscht wurde und eigentlich wieder hinzugefügt werden sollte. Daher habe ich jetzt den ":" wieder eingefügt.